aaq/views.py

```diff
@@ -0,0 +1,116 @@
+import json
+import logging
+
+import requests
+from django.conf import settings
+from rest_framework import status
+from rest_framework.decorators import api_view, renderer_classes
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
+
+from aaq.serializers import InboundCheckSerializer, UrgencyCheckSerializer
+
+from .tasks import send_feedback_task
+
+logger = logging.getLogger(__name__)
+
+
+@api_view(("POST",))
+@renderer_classes((JSONRenderer,))
+def get_first_page(request, *args, **kwargs):
+
+    serializer = InboundCheckSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+
+    question = serializer.validated_data["question"]
+
+    url = f"{settings.AAQ_CORE_API_URL}/inbound/check"
+    payload = {"text_to_match": f"{question}"}
+    headers = {
+        "Authorization": settings.AAQ_CORE_INBOUND_CHECK_TOKEN,
+        "Content-Type": "application/json",
+    }
+
+    response = requests.request("POST", url, json=payload, headers=headers)
+    top_responses = response.json()["top_responses"]
+
+    json_msg = {}
+    body_content = {}
+    message_titles = []
+
+    counter = 0
+    for id, title, content in top_responses:
```

Comment:
You can use `enumerate` here to avoid having to manually increment `counter`

Reply:
Done